src/test/ui/nll/closure-requirements/propagate-approximated-fail-no-postdom.stderr

```diff
@@ -1,28 +1,28 @@
 warning: not reporting region error due to nll
   --> $DIR/propagate-approximated-fail-no-postdom.rs:55:21
    |
-LL |             let p = x.get();
+LL |             let p = x.get(); //~ ERROR
    |                     ^^^^^^^
 
 error: unsatisfied lifetime constraints
-  --> $DIR/propagate-approximated-fail-no-postdom.rs:57:13
+  --> $DIR/propagate-approximated-fail-no-postdom.rs:55:21
    |
 LL |         |_outlives1, _outlives2, _outlives3, x, y| {
    |          ----------              ---------- lifetime `'2` appears in this argument
    |          |
    |          lifetime `'1` appears in this argument
-...
-LL |             demand_y(x, y, p) //~ ERROR
-   |             ^^^^^^^^^^^^^^^^^ argument requires that `'1` must outlive `'2`
+LL |             // Only works if 'x: 'y:
+LL |             let p = x.get(); //~ ERROR
```

Comment:
At first blush I was going to ask whether we were better off with the previous diagnostic, where the call to `demand_y(..)` is highlighted, since in my mind the relevant constraint on the relationship between `'x` and `'y` is "introduced" by that call, and so it is arguably the important thing to highlight.

But after some review of the test and further thought, I think I've convinced myself that this new diagnostic is also good, and perhaps better, since it is telling us where the '`x` is flowing into a variable that will eventually need to satisfy `'y`.

So at this point I'm more just writing this comment to ask you whether you have any thoughts on the relative value of the two spans. Should we be trying to highlight both of them in some manner, in the long term, perhaps?

Reply:
Eventually, I had planned to highlight both spans, yes. I wanted to go through and label all the "significant" points along the path, basically -- but right now we just pick one (and apparently our choice is somewhat influenced here by the order of the search).